### ISSUE TITLE
QA to add 'please test' labels

### ISSUE THREAD

**Issue by jay-meister** [dwyl/contributing#68]
>Finally someone on the team (usually the developer who created the PR) applies the please-test label to any issues mentioned in the PR so that the Product Owner and/or User Testing Team are aware that the feature is ready to test.

Given that the QA has opened all issues that the PR has referenced as part of their PR review, I thought it would make most sense for that person to add the 'please test' label to the relevant issues rather than the developer.

**Comment by nelsonic** [comment]
@JMurphyWeb good point. we will get "***dwylbot***" to do this for us! 😉

**Comment by jay-meister** [comment]
Yeah i guess it is up for discussion, having never QA'd before, my angle is definitely coming from the developers side.

**Comment by nelsonic** [comment]
@JMurphyWeb the question you've _always_ got to ask yourself is: "_whose time is **scarcest**_?" 😉 
When you have to QA 3 projects and still get your _own_ work done your time _vanishes_. 😢 
the solution is really to Automate this task, so we're going to have to write a "Bot" (_specifically "DWYLBot"_) that will do the _mundane_ things like applying "_workflow labels_" to issues/PRs for us. 🤖  #SolveProblemsOnceAndForAllByAutomating!!

> meanwhile, until **dwylbot** is active, I would say that _for now_ it should be part of the Dev's task(s) to apply the `please-test` label. 🏷

**Comment by jay-meister** [comment]
@nelsonic fair enough. I would then ask the question that if a QA doesn't have time to add a please test label to an issue that is already open as part of their work flow, then they are possibly being squeezed a bit too tight?

**Comment by SimonLab** [comment]
link to https://github.com/dwyl/labels/issues/48#issuecomment-277114855 (no automatic please-test label)

**Comment by ghost** [comment]
Is it the QA that requests the developer to test or the developer that requests the QA to test?

Former > QA applies `please-test` label
Latter > Dev applies `please-test` label

Surely?

**Comment by SimonLab** [comment]
For me:
- When the QA (or developer) add the "please-test" label to an issue it means that the product owner can now test the feature
- When a developer ask a QA to test it means adding the "awaiting-review" to the PR
- A QA asking a developer to test is a PR that has been rejected and needs to be updated
